src/compiler/types/lines_2000_4000.rs

```diff
@@ -17,24 +18,34 @@ use local_macros::ast_type;
 pub struct BinaryExpression {
     pub _node: BaseNode,
     pub left: Rc<Node>,
-    pub operator_token: Box<Node>,
+    pub operator_token: Rc<Node>,
     pub right: Rc<Node>,
+    cached_literal_kind: Cell<Option<SyntaxKind>>,
```

Comment:
This only applies to `BinaryPlusExpression` but seemed to make sense to put it on `BinaryExpression`